modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb

```diff
@@ -16,14 +16,20 @@ class PoaFormBuilderJob
     # it queues a job to update the POA code in BGS, as well.
     #
     # @param power_of_attorney_id [String] Unique identifier of the submitted POA
-    def perform(power_of_attorney_id)
+    def perform(power_of_attorney_id, participant_id)
```

Comment:
Unfortunately, this job is referenced in other places and those references would also need to be updated to account for this new arg.  But I think there might be an easier way of getting access to this value without needing to pass a new arg.  Since you're OOO, I'm going to try my hand at it.

Reply:
[Here is the PR](https://github.com/department-of-veterans-affairs/vets-api/pull/9232) that I've proposed.